cmd/dosa/main_test.go

```diff
@@ -57,11 +57,10 @@ func TestHostOptionButNothingElse(t *testing.T) {
 func TestInvalidHost(t *testing.T) {
 	c := StartCapture()
 	exit = func(r int) {}
-	os.Args = []string{"dosa", "--host", "invalid-hostname.", "schema", "check", "--prefix", "foo", "../../testentity"}
+	os.Args = []string{"dosa", "--host", "", "schema", "check", "--prefix", "foo", "../../testentity"}
```

Comment:
An update to YARPC removed this check. Our code only checks to see if it's an empty string.